### PR TITLE
Share collections

### PR DESCRIPTION
**Pull request by piuccio**
hides the implementation details of collection backfill inside facia scala client

@janua

### CODE REVIEW

File: facia-press/app/frontpress/FapiFrontPress.scala [hunk 1]
```diff
@@ -124,15 +124,8 @@ trait FapiFrontPress extends QueryDefaults with Logging with ExecutionContexts {
...
-    collection
-      .collectionConfig
-      .apiQuery
-      .map { query =>
-        FAPI.backfill(query, collection, searchApiQuery, itemApiQuery)
-      }
-      .getOrElse {
-        Response.Right(Nil)
-      }.map(_.map(PressedContent.make))
+    FAPI.backfillFromConfig(collection, searchApiQuery, itemApiQuery)
```
Comment (rich-nguyen) [inline facia-press/app/frontpress/FapiFrontPress.scala]: why does `backfillFromConfig` take a `Collection`, and not the config in the collection? Should it be called `backfillFromCollection`?

**Comment by rich-nguyen** [comment]
:+1:

File: facia-press/app/frontpress/FapiFrontPress.scala (hunk 1 above)
Comment (janua) [inline facia-press/app/frontpress/FapiFrontPress.scala]: You should leave this name but use `collection.collectionConfig` instead; backfill definitely comes from config. You will have to change `facia-scala-client` and release it again.

**Comment by piuccio** [comment]
changes made, it now takes a collection config.

**Pull request merged**